packages/components/Menu/src/MenuDivider/MenuDivider.types.ts

```diff
@@ -6,14 +6,20 @@ import type { IBackgroundColorTokens } from '@fluentui-react-native/tokens';
 
 export const menuDividerName = 'MenuDivider';
 
+export const DividerInsetSizes = [0, 16, 56, 68, 72, 108] as const;
+export type DividerInsetSize = (typeof DividerInsetSizes)[number];
```

Comment:
nit: should probably have "Menu" prefix so it doesn't clash with Divider types.

Reply:
Add Menu Prefix.